lib/rex/parser/netsarang.rb

```diff
@@ -0,0 +1,91 @@
+require 'rex/parser/ini'
+
+module Rex
+  module Parser
+    module NetSarang
+      class NetSarangCrypto
+        attr_accessor :version
+        attr_accessor :username
+        attr_accessor :sid
+        attr_accessor :master_password
+        attr_accessor :key
+
+        # new(type, version, username, sid, master_password = nil)
+        #
+        # === Argument
+        # === Options
+        # :type              :: [Enum]   xshell or xftp.
+        # :version           :: [String] Specify version of session file.
+        # :username          :: [String] Specify username. This parameter will be used if version > 5.2.
+        # :sid               :: [String] Specify SID. This parameter will be used if version >= 5.1.
+        # :master_password   :: [String] Specify user's master password.
```

Comment:
A quick note: code documentation in Metasploit is in the [YARD format](https://rubydoc.info/gems/yard/file/docs/GettingStarted.md) (like https://github.com/rapid7/metasploit-framework/blob/1acb5ac18aa62aa3e00314e815531ca11b1bc8da/lib/msf/core/payload_set.rb#L200-L214). Thanks for adding code docs!